### PR TITLE
Build assets before checking they're unchanged

### PR DESCRIPTION
**Pull request by samwilson**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,10 +20,15 @@ jobs:
             docker-compose exec web composer lint
             docker-compose exec assets npm run lint
 
+      - run:
+          name: Build assets
+          command: |
+            docker-compose exec assets npm run assets
+            git status
+            git status | grep "nothing to commit, working tree clean"
```
Comment (MusikAnimal) [inline .circleci/config.yml]: just curious, why do we need to run `git status` twice?

Comment (samwilson) [reply .circleci/config.yml]: It's so when the 2nd one fails we can see above what files it's failing on. Not strictly required, but useful when trying to debug failures.

Comment (MusikAnimal) [reply .circleci/config.yml]: Ah, clever.

Comment (samwilson) [reply .circleci/config.yml]: The current failure is a case in point: https://circleci.com/gh/wikimedia/svgtranslate/140

**Comment by samwilson** [comment]
This is passing now.

**Pull request merged**